hardhat.config.ts

```diff
@@ -1,10 +1,26 @@
 import 'hardhat-typechain'
 import '@nomiclabs/hardhat-ethers'
 import '@nomicfoundation/hardhat-chai-matchers'
+import '@nomicfoundation/hardhat-foundry'
 import dotenv from 'dotenv'
 dotenv.config()
 
 const DEFAULT_COMPILER_SETTINGS = {
+  version: '0.8.26',
+  settings: {
+    viaIR: true,
+    evmVersion: 'cancun',
+    optimizer: {
+      enabled: true,
+      runs: 1_000_000,
+    },
+    metadata: {
+      bytecodeHash: 'none',
+    },
+  },
+}
+
+const PERMIT2_COMPILER_SETTINGS = {
```

Comment:
do we ever really deploy Permit2 from this repo?? Some repos just have the static bytecode instead of re-compiling. No strong opinions, but just a thought...

Reply:
this is solved in #347 where i rip out all the permit2 stuff